README.md

```diff
@@ -102,9 +102,47 @@ Clone the repo to get started and follow these steps:
 The docker-compose setup comes with two pre-built Grafana dashboards. One for listing the discrete test runs as a list, and the other for visualizing the results of a specific test run.
 >Note: The dashboards work with the Native Histogram mapping so it is required to enable it.
 
-### Test result dashboard
-![Prometheus dashboard of k6 test result](./images/prometheus-dashboard-k6-test-result.png)
 
 ### Test list dashboard
 ![Prometheus dashboard of k6 test runs](./images/prometheus-dashboard-k6-test-runs.png)
 
+>Note: This dashboard depends on the use of testid tag
+### Test result dashboard
+
+![Prometheus dashboard of k6 test result](./images/prometheus-dashboard-k6-test-result.png)
```

Comment:
We may have a smaller screenshot, it is not required to preview the entire screen. We can just use the top part. We already preview all the sections in detail after.